src/frontend/web_application/src/components/MessageDate/style.scss

```diff
@@ -0,0 +1,13 @@
+@import '../../styles/common';
+@import '../../styles/vendor/bootstrap_foundation-sites';
+
+.m-message-date {
+  &__time { display: block; }
+
+  @include breakpoint(medium) {
+    &__time {
+      display: inline;
+      margin-left: $co-component__spacing;
```

Comment:
don't you think this spacing variable must be used for components (I mean divs, blocks, sections ...)  not for word spacing ?